sh/build-database.sh

```diff
@@ -33,11 +33,11 @@ mysql -u $DB_USER -p$DB_PASS $DB_NAME < server/models/procedures.sql &> /dev/nul
 mysql -u $DB_USER -p$DB_PASS $DB_NAME < server/models/admin.sql &> /dev/null
 
 echo "[build] default data"
-mysql -u $DB_USER -p$DB_PASS $DB_NAME < server/models/test/icd10.sql &> /dev/null
+mysql -u $DB_USER -p$DB_PASS $DB_NAME < test/icd10.sql &> /dev/null
```

Comment:
icd10.sql should remain where it is - they are medical codes that we will always ship with the application.